src/gui/sidebar/previews/base/SidebarPreviewBase.cpp

```diff
@@ -128,6 +128,6 @@ auto SidebarPreviewBase::scrollToPreview(SidebarPreviewBase* sidebar) -> bool {
     return false;
 }
 
-void SidebarPreviewBase::pageDeleted(int page) {}
+void SidebarPreviewBase::pageDeleted(size_t page) {}
```

Comment:
For those really large documents with more than 2147483647 pages? 😉 
But leave it there, it doesn't hurt.

Reply:
I was getting a warning in the mac build CI check :)